models/aeif_cond_beta_multisynapse.h

```diff
@@ -73,7 +73,7 @@ modeled by a beta function.
 
 It allows an arbitrary number of synaptic
 rise time and decay time constants. Synaptic conductance is modeled by a
-beta function, as described by A. Roth and M.C.W. van Rossum
+beta function, as described by A. Roth and M. C. W. van Rossum
```

Comment:
typically our style is without spaces here, I believe (same as for citations)

Reply:
I went back to the old style.